lib/auxlib.c

```diff
@@ -102,16 +102,20 @@ void config_load_free(struct rtpengine_common_config *cconfig) {
 }
 
 static void free_gkeyfile(GKeyFile **k) {
-	g_key_file_free(*k);
+	if (k && *k)
+		g_key_file_free(*k);
```

Comment:
These shouldn't be necessary as the pointer in an auto-cleanup function should always be non-null. But doesn't hurt either of course.